### DISCUSSION TITLE
Please add *tiktokcdn-eu.com to the blocked service feature

### DISCUSSION THREAD

**Discussion by digitalSeb** [AdguardTeam/AdGuardHome#6561] [Ideas]
As the title says, please add *tiktokcdn-eu.com to the blocked service feature. Right now it doenst block that

**Comment by ainar-g** [comment]
<https://github.com/AdguardTeam/HostlistsRegistry> is the repository hosting the service information.  If you think that something is missing, please file an issue there.  AdGuard Home's databases are updated during Beta and Final releases.
